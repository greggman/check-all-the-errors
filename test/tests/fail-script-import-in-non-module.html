<script>
import * as foo from './resources/module.js';  // eslint-disable-line
</script>
