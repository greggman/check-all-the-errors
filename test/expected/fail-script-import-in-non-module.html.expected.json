{
  "numErrors": 1,
  "pages": {
    "http://localhost:8090/fail-script-import-in-non-module.html": {
      "status": 200,
      "errors": [
        {
          "type": "pageerror",
          "error": "Error: SyntaxError: Cannot use import statement outside a module"
        }
      ]
    }
  },
  "responses": [
    {
      "href": "http://localhost:8090/fail-script-import-in-non-module.html",
      "resourceHref": "http://localhost:8090/fail-script-import-in-non-module.html",
      "status": 200
    }
  ]
}